src/PostgREST/Config.hs

```diff
@@ -172,6 +174,7 @@ readOptions = do
         |## (use "@filename" to load from separate file)
         |# jwt-secret = "foo"
         |# secret-is-base64 = false
+        |# jwt-aud = "your_audience_claim"
```

Comment:
We'll want to add a new variable to the Docker setup as well, much like how PGRST_JWT_SECRET is handled in the [Dockerfile](https://github.com/begriffs/postgrest/blob/master/docker/Dockerfile#L34) and [postgrest.conf](https://github.com/begriffs/postgrest/blob/master/docker/postgrest.conf#L10). Those two files are used for building our images on Dockerhub.

Reply:
Good catch! I updated those files as well as app.json.